File: back/repository/src/main/java/org/webtree/mydata/repository/PersonalDataRepository.java
package org.webtree.mydata.repository;

import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;
import org.webtree.mydata.domain.PersonalData;

@Repository
public interface PersonalDataRepository extends ReactiveMongoRepository<PersonalData, String> {
}

File: api/src/main/kotlin/org/webtree/mydata/api/repository/PersonalDataRepository.kt
package org.webtree.mydata.api.repository

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import org.webtree.mydata.api.domain.PersonalData

@Repository
interface PersonalDataRepository: ReactiveMongoRepository<PersonalData, String>